packages/flutter/lib/src/material/snack_bar_theme.dart

```diff
@@ -115,6 +117,17 @@ class SnackBarThemeData with Diagnosticable {
   /// [SnackBarBehavior.floating].
   final double? width;
 
+  /// Overrides the default value for [SnackBarIcon.icon].
+  ///
+  /// If null, and an [icon] property is included in the snack bar constructor,
+  /// a default 'Close' icon will display.
+  final Widget? icon;
+
+  /// Overrides the default value for [SnackBar.margin].
```

Comment:
Docs for `SnackBar.margin` and `behavior` should be updated accordingly

Reply:
Done, thanks